config/v3_1/types/headers_test.go

```diff
@@ -121,19 +120,19 @@ func TestValidHeadersParse(t *testing.T) {
 	// Valid headers
 	headers := HTTPHeaders{
 		HTTPHeader{
-			Name:  "header1",
+			Name:  "Header1",
```

Comment:
This invalidates the test though.  We want to see if header parsing properly normalizes case.